code-reviews/recipes/Python.md

```diff
@@ -16,14 +16,26 @@ Both `Pylint` and `Flake8` can be configured in VS Code using the VS Code `pytho
 
 ### Flake8
 
-Flake8 is a simple and fast wrapper around [`Pyflakes`](https://github.com/PyCQA/pyflakes) (for detecting coding errors), [`pycodestyle`](https://github.com/PyCQA/pycodestyle) (for pep8) and [`pydocstyle`](https://github.com/PyCQA/pydocstyle) (for [doc strings](https://www.python.org/dev/peps/pep-0257/)).
+Flake8 is a simple and fast wrapper around [`Pyflakes`](https://github.com/PyCQA/pyflakes) (for detecting coding errors) and [`pycodestyle`](https://github.com/PyCQA/pycodestyle) (for pep8).
 
 Install `Flake8`
 
 ```bash
 pip install flake8
 ```
 
+Add an extension for the [`pydocstyle`](https://github.com/PyCQA/pydocstyle) (for [doc strings](https://www.python.org/dev/peps/pep-0257/)) tool to flake8.
+
+```bash
+pip install flake8-docstrings
+```
+
+Add an extension for [`pep8-naming`](https://github.com/PyCQA/pep8-naming) (for [naming conventions](https://www.python.org/dev/peps/pep-0008/#naming-conventions) in pep8) tool to flake8
```

Comment:
Nit: For consistency, end the sentence with punctuation here too.

Reply:
Updated. Good catch.